package.json

```diff
@@ -37,12 +37,19 @@
   "homepage": "https://github.com/mljs/ml-directional-distribution#readme",
   "devDependencies": {
     "@types/jest": "^27.4.0",
+    "@types/node": "^17.0.10",
     "eslint": "^8.7.0",
     "eslint-config-cheminfo-typescript": "^10.3.0",
     "jest": "^27.4.7",
     "prettier": "^2.5.1",
     "rimraf": "^3.0.2",
     "ts-jest": "^27.1.3",
     "typescript": "^4.5.4"
+  },
+  "dependencies": {
+    "ml-array-sum": "^1.1.5",
+    "ml-array-variance": "^1.1.7",
+    "ml-array-xy-covariance": "^0.2.1",
+    "ml-matrix": "^6.8.2"
```

Comment:
Do we really need to depend on ml-matrix ? It is a relatively big library but if there is no obvious alternative we can keep it.